src/api/item.js

```diff
@@ -37,8 +37,9 @@ app.get('/all', (req, res) => {
 });
 
 app.delete('/', (req, res) => {
-    let itemToDelete = ItemStore.getItemByAddress(req.query.itemAddress);
-    let model = ModelStore.getModelByAddress(itemToDelete.modelAddress);
+    if(!req.query.itemAddress) return res.failureJson("Item address required to delete");
+    if(!req.query.modelAddress) return res.failureJson("Model address required to delete");
```

Comment:
Nit: This isn't checking for the correct parameter type, only that it exists / is truthy. What happens if, for example, an empty array is passed for the `modelAddress`?